keps/sig-node/5075-dra-dynamic-provisioning-support/README.md

```diff
@@ -0,0 +1,1055 @@
+
+There are also similar use cases for requesting the device provision based on a specific root or device type in other contexts including [Peer Pods](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/docs/architecture.md) and [Composable Disaggregated Infrastructure (CDI)](https://www.wwt.com/article/an-introduction-to-composable-disaggregated-infrastructure). 
+
+This feature is also beneficial for the other provisonable devices those are not with a scope of [KEP-4815](https://github.com/kubernetes/enhancements/issues/4815). For instance, this feature will be allow reserving memory fraction of virtual GPU in [the AWS virtual GPU device plugin](https://github.com/awslabs/aws-virtual-gpu-device-plugin) via DRA. The number of provision can be limited if needed.
+
+### Goals
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+- Introduce an ability to allocating on-demand provisioned devices via DRA. This should cover the use case of macvlan, ipvlan in DRA driver for CNI and virtual accelerator devices with on-demand memory fraction. 
+- Enhance a capability of secondary networks to dynamically allocate secondary networks based on present availabilities such as bandwidth.
+- Leverage the enhancement of capacity consumption from [KEP-4815](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/4815-dra-partitionable-devices).
+
+### Non-Goals
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+- Define device-specific or vendor-specific attributes.
+- Support network security policy.
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+### User Stories (Optional)
+
+<!--
+Detail the things that people will be able to do if this KEP is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+-->
+
+#### Story 1
+Users can request for secondary networks based on their bandwidth demands without the need to concern about neither device name nor a present available bandwidth of each specific device. DRA scheduler can dynamically select available device based on its availability. 
```

Comment:
Edited for clarity:  A user requests a secondary network based on their bandwidth demands without wanting to specify a device name nor present available bandwidth of each specific device. The DRA scheduler can dynamically select available devices based on their availability.